### ISSUE TITLE
Hi riza this is Harshaa

### ISSUE THREAD

**Issue by Harshaa-99** [rizasimkhana/sql#1]
Hi @rizasimkhana this is Harshaa your guvi learning mate sorry for texting you in this unprofessional way. i've already tried to contact you in linkedin     but i din't get any response. since you are good at the projects which we are doing but iam facing some issues in that. Good to have a contact from an professional like you.. The answers and the projects which you are showing in the class is like mindblowing you know many things also doing the projects like you are professional because you are. i would like to have some discussions from you about our classes and projects, kindly help me with that. this is my number :7871482205

**Comment by rizasimkhana** [comment]
Thanks for contacting me I've accepted your LinkedIn request let's discuss
there if you've any doubts

**Comment by Harshaa-99** [comment]
Hi riza can we connect and have a call

**Comment by rizasimkhana** [comment]
sure but whats the issue may i know

**Comment by Harshaa-99** [comment]
Just a connect and have an friendly convo. Is it okay for u

**Comment by rizasimkhana** [comment]
It's not about how the call is I have a newborn baby so most of time I have to spend with him along the class projects if you text me if you've any issues so I could sort out hope you have a good day

**Comment by Harshaa-99** [comment]
Sorry riza for not understanding your situation I thought u r working like
the me. Just want to talk about how you are doing coding better than us and
making projects faster than us. No problem riza we ll catch up some other
time. We ll be in touch. Also thanks for replying

**Comment by rizasimkhana** [comment]
Thanks man actually I love to help people more I hope to work again in office so joined this course I don't know I'm that better but I do what I know see your projects LinkedIn it's really good

**Comment by Harshaa-99** [comment]
Sorry i dint meant to disturb you just wanna talk about guvi and about the
projects feel free to call me when u r free. Maybe with ur help i can gain
more knowledge. Also nice knowing you riza

**Comment by rizasimkhana** [comment]
It's not disturbance you're my learning mate we share some of knowledge gain more right.find some time to talk to you

**Comment by Harshaa-99** [comment]
Hi riza can u share me the netlify link of the rental project u developed
iam interested

**Comment by rizasimkhana** [comment]
https://fanciful-custard-ad237e.netlify.app/

**Comment by rizasimkhana** [comment]
it is simple  and admin has some mistakes

**Comment by Harshaa-99** [comment]
Thanks riza

**Comment by Harshaa-99** [comment]
Do you have the login password

**Comment by rizasimkhana** [comment]
if you google register and do google login if you do traditional register and just do traditional login

**Comment by rizasimkhana** [comment]
in guvi they give low marks for me they told they cant login but i could login

**Comment by Harshaa-99** [comment]
I can login too but as a user what about admin

**Comment by rizasimkhana** [comment]
can you see user board

**Comment by rizasimkhana** [comment]
all user functionalites are working

**Comment by Harshaa-99** [comment]
Can we have a call in detail

**Comment by Harshaa-99** [comment]
Yes and all are working fine

**Comment by rizasimkhana** [comment]
How much you got for your project

**Comment by Harshaa-99** [comment]
I felt like the sessions are fast and I ve told them to take the react and
express once again so i dint move forward to the project

**Comment by Harshaa-99** [comment]
Have they started the placements for u

**Comment by rizasimkhana** [comment]
No we've to complete 2 final projects

**Comment by Harshaa-99** [comment]
so you have one project left right?

**Comment by rizasimkhana** [comment]
Nope I have to pass this project first they said they couldn't login put 3 marks for me and redo i again resubmitted project let's see

**Comment by Harshaa-99** [comment]
Ok riza

**Comment by Harshaa-99** [comment]
But totally 2 projects right

**Comment by Harshaa-99** [comment]
i can login but it comes a dangerous site

**Comment by rizasimkhana** [comment]
What can I get screenshot of it

**Comment by rizasimkhana** [comment]
its under deployment so only you got this error now it is functional you can use it

**Comment by Harshaa-99** [comment]
now i cant able to login

**Comment by rizasimkhana** [comment]
Yeah because I cleared db just login with google again

**Comment by rizasimkhana** [comment]
Have you submitted project

**Comment by Harshaa-99** [comment]
No not yet.

**Comment by rizasimkhana** [comment]
Did you got any mail from guvi

**Comment by Harshaa-99** [comment]
About what

**Comment by rizasimkhana** [comment]
Like projects or something placement team performance message

**Comment by Harshaa-99** [comment]
No I dint get anything what about u did u get any mail

**Comment by rizasimkhana** [comment]
No mail from guvi

**Comment by rizasimkhana** [comment]
Did you have any mail for mock interview

**Comment by Harshaa-99** [comment]
No I am not on placement phase yet

**Comment by rizasimkhana** [comment]
do you got any mail from guvi regarding problem sloving

**Comment by Harshaa-99** [comment]
No how did your mock interview went and did u get placed

**Comment by rizasimkhana** [comment]
how much you got

**Comment by rizasimkhana** [comment]
no i got 42

**Comment by Harshaa-99** [comment]
Glad to hear it! also i told you that i dint get forward to placement phase
will let you know if i have any updates . what about the placement did you
attend any interviews.

**Comment by Harshaa-99** [comment]
Hi riza this is a personal request since I have already changed my batch
and our class recordings are gone. my new mentor is not as good as venkat I
am trying to download the zoom recordings when we had classes with venkat
can you share your zen class credentials so that I will download the
recordings and give it back to you. I know this is a very big thing please
help me with this and I won't misuse it anywhere and you are the only
learning mate I have so kindly help me I will log out as soon as I download

**Comment by rizasimkhana** [comment]
sure riza.simkhanas@gmail.com Riza@0303

**Comment by rizasimkhana** [comment]
take your own time and download it i don't mind

**Comment by Harshaa-99** [comment]
Thanks a lot riza

**Comment by Harshaa-99** [comment]
Hi riza any updates on  placements🥲

**Comment by rizasimkhana** [comment]
nope

**Comment by rizasimkhana** [comment]
do you have any updates regarding placements

**Comment by Harshaa-99** [comment]
No updates yet. Did you ask what is the issue because u r the top performer
in our batch how didn't you get Andy placement calls

**Comment by Harshaa-99** [comment]
How don't you get any placement calls till now it's been two months right

**Comment by rizasimkhana** [comment]
who told im the top performer

**Comment by rizasimkhana** [comment]
how is your new classes

**Comment by Harshaa-99** [comment]
I ve changed to tamil class and manivannan takes the classes but he is like
romba obi adikura maari theriyuthu no one is like venkat our best mentor

**Comment by rizasimkhana** [comment]
yes tamil batches are like that

**Comment by rizasimkhana** [comment]
problem sloving session in tamil also like that

**Comment by Harshaa-99** [comment]
Did you ask about the placement

**Comment by rizasimkhana** [comment]
yes now only they conduct placement eligiblity test

**Comment by Harshaa-99** [comment]
You are already eligible right??

**Comment by rizasimkhana** [comment]
I Dont know they conduct test

**Comment by Harshaa-99** [comment]
Hi riza any updates about the placement did you get placed

**Comment by rizasimkhana** [comment]
No training session only I'm in .i submit protofolio and resume to guvi they review and tell how they look.

**Comment by Harshaa-99** [comment]
Don't you think that they are cheating its been so long you finished the
course and also you are a top scorer. Are they saying some reasons to waste
time. I think so they are not good at placements. What was your view on this

**Comment by rizasimkhana** [comment]
i don't think so i cleared pte exam they conduct now im in placement session they alotted 3 months of time preoid there is some process going on after clear all this resume protofolio and mock interview in between we get a job i think so. they send a company in this process but i don't like that company job role was also different.how did you got certificate i saw it on linkedin

**Comment by rizasimkhana** [comment]
i got certificate

**Comment by Harshaa-99** [comment]
No I dint get the certificate what post on linkedin can you send a
screenshot also we dint get connected on linkedin

**Comment by rizasimkhana** [comment]
![Image](https://github.com/user-attachments/assets/e869717e-8fba-4b79-8a4e-39ba133d7135)

**Comment by Harshaa-99** [comment]
My name is harshaa not harish

**Comment by Harshaa-99** [comment]
😂

harshaa reacted via Gmail
<https://www.google.com/gmail/about/?utm_source=gmail-in-product&utm_medium=et&utm_campaign=emojireactionemail#app>

**Comment by Harshaa-99** [comment]
https://www.linkedin.com/in/harshaa?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

**Comment by rizasimkhana** [comment]
ohh kk

**Comment by rizasimkhana** [comment]
whats your native

**Comment by Harshaa-99** [comment]
Chengalpattu next to chennai

**Comment by rizasimkhana** [comment]
ok currently are you working

**Comment by Harshaa-99** [comment]
Yes what about you

**Comment by rizasimkhana** [comment]
may i know which company
